lib/src/ui.dart

```diff
@@ -10,29 +10,107 @@ import 'network_event.dart';
 import 'network_logger.dart';
 
 /// Overlay for [NetworkLoggerButton].
-class NetworkLoggerOverlay extends StatelessWidget {
-  NetworkLoggerOverlay._({Key? key}) : super(key: key);
+class NetworkLoggerOverlay extends StatefulWidget {
+  NetworkLoggerOverlay._({this.right, this.bottom, Key? key}) : super(key: key);
+
+  double? bottom;
+  double? right;
```

Comment:
```suggestion
  NetworkLoggerOverlay._({this.right = 30, this.bottom = 30, Key? key}) : super(key: key);

  final double bottom;
  final double right;
```

Reply:
No argument is required, so I don't think this would be a good change and also it's difficult to refactor it to be like that because then .attachTo() parameters can not be null and must be provided so they would become required which we don't want

Reply:
> become required which we don't want

Not really we can provide fallback value (`int right = 30, int bottom = 30`) there too to make them optional.

Reply:
@themisir  Do you like it like [this](https://github.com/themisir/flutter-network-logger/pull/9/commits/c11e830358d2750c8b177ce90eae5008bf2148b8)